Clinical-Genomics/scout
# hotfix version 4.6.1

This PR fixes the bug described in #1268

**How to test**:
This bug is reproducible and testable by loading the cancer case in demo data:
1. To reproduce: from root folder of a local scout repo (being in master branch)
```
scout --demo load case scout/demo/cancer.load_config.yaml
```
 You will observe the error described in  #1268 ('father' key is missing in parse case function).


2. git checkout to this fix branch:
```
git checkout  release_4.6.1
```
Repeat the load case command described above and the upload of the case should be successful.

**Expected outcome**:
Case should be correctly inserted

**Review:**
- [x] code approved by Daniel Nilsson
- [x] tests executed by Chiara Rasi
